pvlib/solarposition.py

```diff
@@ -328,25 +328,26 @@ def spa_python(time, latitude, longitude,
     -------
     DataFrame
         The DataFrame will have the following columns:
-        apparent_zenith (degrees),
-        zenith (degrees),
-        apparent_elevation (degrees),
-        elevation (degrees),
-        azimuth (degrees),
-        equation_of_time (minutes).
 
+        - apparent_zenith (degrees),
+        - zenith (degrees),
+        - apparent_elevation (degrees),
+        - elevation (degrees),
+        - azimuth (degrees),
+        - equation_of_time (minutes).
 
     References
     ----------
     .. [1] I. Reda and A. Andreas, Solar position algorithm for solar
        radiation applications. Solar Energy, vol. 76, no. 5, pp. 577-589, 2004.
+       :doi:`10.1016/j.solener.2003.12.003`.
 
     .. [2] I. Reda and A. Andreas, Corrigendum to Solar position algorithm for
        solar radiation applications. Solar Energy, vol. 81, no. 6, p. 838,
-       2007.
+       2007. :doi:`10.1016/j.solener.2007.01.003`.
 
-    .. [3] USNO delta T:
-       https://maia.usno.navy.mil/products/deltaT
+    .. [3] `USNO delta T:
```

Comment:
```suggestion
    .. [3] `U.S. Naval Observatory, delta T
```
Optional/suggestion: could we write this out in full? Seems nicer somehow